requirements/testing.txt

```diff
@@ -2,3 +2,4 @@ coverage
 pytest
 pytest-cov
 pytest-django
+git+https://github.com/django-crispy-forms/crispy-bootstrap4
```

Comment:
We need multiple template packs available to test the overriding of template packs.